concepts/match-basics/about.md

```diff
@@ -1 +1,121 @@
 # Match Basics
+
+The `match` expression allows for easy comparison of a value against a series of patterns and afterwards run the code based on which pattern matches. Literal values, variable names, wildcards, and many other things are what makes up a pattern.
+
+The examples below show the `match` syntax:
+
+```rust
+match enum_var {
+    variant_0(a, b, c) => {/* code */}
+    variant_1(_) => {/* code */}
+    ...
+    variant_k(a, c) => {/* code */}
+}
+
+match felt_var {
+    0 => { /* code */ }
+    _ => { /* code */ }
+}
+```
+
+The `variant_0`, `variant_1`, `...`, and `variant_k` are all variants of an enum data type. Where `enum_var` could be any instance of the enum, after matching, executes the code in the curly braces. While the second example is the usage of the match patterning with a `felt_var` felt252 with arm patterns of value `0` and wildcard, `_`.
+
+Following the format above, we can write a function that takes in an unknown color parameter and return its true color.
+
+```rust
+enum Color {
```

Comment:
This is a good example! You can use this above instead of the enum example with placeholders.
You can then delete the sentence above (the "Following the format..." one).

Just make sure you use valid Cairo code, i.e. one that compiles